massgov/mayflower
# Feature/dp 14173 service page redesign

Any PRs being created needs a changelog.txt file before being merged into dev. See: [Change Log Instructions](https://github.com/massgov/mayflower/blob/dev/docs/change-log-instructions.md)


## Description


## Related Issue / Ticket

- [JIRA issue](https://jira.mass.gov/browse/DP-14173)
- [Github issue](https://github.com/massgov/mayflower/tree/feature/DP-14173-service-page-redesign)

## Steps to Test


1. go to: http://localhost:3000/?p=templates-services-flexible-link-groups
2. verify that the design of this page matches the layout described in the ticket and in the invision comp at https://eotss.invisionapp.com/share/JFS2B2N8VWX#/screens/364335348

## Screenshots
N/A


## Additional Notes:
N/A

#### Impacted Areas in Application


* link lists can now add download links
* comp heading can now hide the underline
* new icon added for links going to login pages (key)
* adds new component for service pages with flexible link groups

#### @TODO


*

#### Today I learned...
